Library/Formula/libu2f-host.rb

```diff
@@ -0,0 +1,20 @@
+class Libu2fHost < Formula
+  desc "Libu2f-host implements the host-side of the U2F protocol. "
+  homepage "https://developers.yubico.com/libu2f-host/"
+  url "https://developers.yubico.com/libu2f-host/Releases/libu2f-host-1.0.0.tar.xz"
+  sha256 "18c56b9b5cfea2566925bba45b25a4e20b3ef8696905d8f2a06116316e164374"
+
+  depends_on "pkg-config" => :build
+  depends_on "hidapi" => :build
+  depends_on "json-c" => :build
+
+  def install
+    args = ["--prefix=#{prefix}"]
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/u2f-host", "--version"
```

Comment:
I could...but it would require the tester to insert a U2F USB key since these commands don't work without one. Not sure if these tests can be interactive.